.github/workflows/t3000-model-perf-tests.yaml

```diff
@@ -61,7 +61,7 @@ jobs:
         run: tar -xvf ttm_${{ matrix.test-group.arch }}.tar
       - uses: ./.github/actions/install-python-deps
       - name: Run model perf regression tests
-        timeout-minutes: 60
+        timeout-minutes: 120
```

Comment:
is this big of a jump needed?

Reply:
No, I decreased it to 75 mins which should be enough time accounting for potential hangs